upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -270,6 +270,11 @@ If the following command does not complete successfully, check if the `TOKEN` en
    [Upgrade Troubleshooting](README.md#relevant-troubleshooting-links-for-upgrade-related-issues).
    If the failure persists, then open a support ticket for guidance before proceeding.
 
+   `prerequisites.sh` clears the existing CFS configuration for each Management node. As each
+   Management node is upgraded, documentation will refer to the CFS configuration that should be
+   assigned to the node at that time. If any Management node is unexpectedly rebooted prior to this,
```

Comment:
```suggestion
   assigned to the node at that time. If any Management node is unexpectedly rebooted prior to the end of the CSM upgrade,
```
```suggestion
   assigned to the node at that time. If any Management node is unexpectedly rebooted prior to this,
```

Reply:
The timeframe when a node would be rebooted and not run CFS node personalization is after ```prerequisites.sh``` runs and
- worker nodes: before the new instructions in Stage 2 tell the admin how to put the initial/critical CFS configuration back in place
- storage and master nodes: until the end of the CSM update

Given that, I am not sure the suggestion is better than the original text.  Thoughts?

Reply:
"this" does imply a more immediate context

Reply:
How about my updated wording?